aboutPage/about.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../reset.css">
+    <link rel="stylesheet" href="./aboutStyles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Chango&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/434da06752.js" crossorigin="anonymous"></script>
+    <title>About Trims</title>
+</head>
+<body>
+  <div class="container">
+    <header>
+      <div class="logo">
+        <i class="fas fa-cut fa-5x fa-rotate-90"></i><br>
+        <span id="trims">Trims</span>
+      </div>
+      <nav class="main-nav">
+        <a href="../index.html">Home</a>
+        <a id="selected" href="">About</a>
+        <a href="../imageGallery/imageGallery.html">Gallery</a>
+        <a href="../blogPage/blog.html">Blog</a>
+        <a href="../contactPage/contact.html">Contact</a>
+      </nav>
+    </header>
+    <main class="content">
+      <h1>Trims Started</h1>
+      <img id="shopPhoto" src="../images/barberShop.jpg" alt="barber shop with brick wall and empty chairs">
+      <article>
+        <p>
+        <h3>Hair Cuts for Everyone</h3>
+        Our website templates are created with inspiration, checked for quality and originality and meticulously sliced and coded. What’s more, they’re absolutely free! You can do a lot with them. You can modify them. You can use them to design websites for clients, so long as you agree with the Terms of Use. You can even remove all our links if you want to.
```

Comment:
we usually don't nest h1/h2/h3/h4/h5/h6 tags inside a p tag. H3 is a heading tag so it labels a title of something. p tag is a paragraph which will go under a title. no need to change it now. might be too mess to move things around but just know for the future!

Reply:
ahh silly me. got it!